src/test/kotlin/no/skatteetaten/aurora/boober/feature/DeploymentConfigFeatureTest.kt

```diff
@@ -230,4 +230,26 @@ class DeploymentConfigFeatureTest : AbstractFeatureTest() {
 
         assertThat(dcResource).auroraResourceMatchesFile("changed-dc-template.json")
     }
+
+    @Test
+    fun `modify dc add nodeSelectors`() {
```

Comment:
Burde du ha en test hvor du prøver å legge på affiliation til en annen som nodeSelector, det vil da være for fremtidig tukling. I tillegg navngi testen med noe som vil forklare at dette må bestå

Reply:
jeg legger inn en test for dette ja